spec/features/finance/npq/course_payment_breakdown_spec.rb

```diff
@@ -305,19 +303,31 @@ def statement_declarations_per_contract(contract)
   end
 
   def total_starts
-    statement_declarations.started.count
+    statement
```

Comment:
these are still bad as we are asserting against a moving target but i'm not fixing in this PR

Reply:
agreed that's not ideal we should add that to let(:expected_(started|retaine|completed)_count